docs/url.md

```diff
@@ -94,6 +94,29 @@ class ShowUsers extends Component
 }
 ```
 
+### Nullable properties
+
+By default, if a page is loaded with an empty query string entry like `?search=`, Livewire will treat that value as an empty string. In many cases, this is expected, however there are times when you want `?search=` to be treated as `null`.
+
+In these cases, you can use a nullable typehint like so:
+
+```php
+use Livewire\Attributes\Url;
+use Livewire\Component;
+
+class ShowUsers extends Component
+{
+    #[Url]
+    public ?string $search; // [tl! highlight]
```

Comment:
I'm only documented the nullable typehint for now as I think `nullable` is unergonomic and most times, the typehint is simple to add.